app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,12 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = Post.all
+    @posts = Post.where(author_id: @user)
   end
```

Comment:
- [ ] You are yet to eliminate the N + 1 query problem for the Post index. You can do that by refactoring to this:

  ```
  def index
    @user = User.find(params[:user_id])
    @posts = @user.posts.includes(:comments)
  end
  ```

Reply:
done